plugins/quickcl.py

```diff
@@ -179,7 +182,12 @@ def __init__(self, microscope, main_app):
         self.light = main_data.light
         self.ccd = main_data.ccd
 
-        self.filename = model.StringVA("", setter=self._set_filename)
+        self.conf = get_acqui_conf()
+        self.conf.last_extension = '.png'
+        self.filename = model.StringVA(create_filename(self.conf.last_path,
```

Comment:
No need to set a beautiful filename at init, as it will be reset when opening the window. => just set it to `""`.